bin/physlr-make

```diff
@@ -1134,12 +1134,12 @@ g=$(shell awk '{x += $$2} END{print x}' $(name)/$(ref).fa.fai)
 	$(time) $(python) $(bin)/physlr molecules -V$V -t$t --separation-strategy=bc+louvain $< >$@
 
 # Make a vertex-induced subgraph
-%.subgraph.tsv: %.tsv
+%.subgraph-d$d.tsv: %.tsv
```

Comment:
Could we add a default setting for `d` at the top of the Makefile, with the other variable declarations? Just so we can have a comment there as well reminding our future selves what this parameter is.

Reply:
Sure, will do

Reply:
done